### PR TITLE
✨ Support New Awesome Twilight Components🦾

### PR DESCRIPTION
**Pull request by eslamoo**
What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)

* Support `salla-product-card`, `salla-products-list` and `salla-products-slider`

What is the current behaviour? (You can also link to an open issue here)

* 

What is the new behaviour? (You can also link to the ticket here)

* 

Does this PR introduce a breaking change?

* 

Screenshots (If appropriate)

*

### CODE REVIEW

File: src/views/components/home/latest-products.twig [hunk 1]
```diff
@@ -7,14 +7,7 @@
...
-    {% if products|length %}
-        <salla-infinite-scroll class="gird-view grid grid-cols-2 md:grid-cols-3 lg:grid-cols-4 gap-2.5 sm:gap-4 md:gap-6 mb-6"
-                               next-page="{{ products.next_page }}"  autoload>
-            {% for product in products %}
-                {% include 'pages.partials.product.card' %}
-            {% endfor %}
-        </salla-infinite-scroll>
-    {% endif %}
+    
+    <salla-products-list></salla-products-list>
```
Comment (salkhwlani) [inline src/views/components/home/latest-products.twig:12]:
```html
<salla-products-list source="latest"></salla-products-list>
```

File: src/views/pages/landing-page.twig [hunk 2]
```diff
@@ -70,15 +70,18 @@
...
             <section class="container overflow-hidden {{landing.products|length <= 3 and not landing.is_slider ? 'center-products' : ''}}">
-                {% set products=landing.products %}
-                {% set hide_add_btn=true %}
-                {% set show_quantity=landing.show_quantity %}
-                {% set title=trans('pages.offer.included_products') %}
-
+                {% set is_vertical = theme.settings.get('vertical_fixed_products', true) %}
                 {% if landing.is_slider %}
-                    {% include 'components.home.products-slider' %}
+                    <salla-products-slider block-title="{{trans('pages.offer.included_products')}}" display-all-url></salla-products-slid
```
Comment (salkhwlani) [inline src/views/pages/landing-page.twig:75]:
```html
<salla-products-list source="landing-page"></salla-products-list>
```

File: src/views/pages/landing-page.twig [hunk 3]
```diff
@@ -70,15 +70,18 @@
...
                 {% else %}
-                    {% include 'components.home.fixed-products' %}
+                    <section class="s-block">
+                        <div class="s-block__title">
+                            <div class="right-side">
+                            <h2>{{ trans('pages.offer.included_products') }}</h2>
+                            </div>
+                        </div>
+                        <salla-products-list class="{{is_vertical ? 'vertical-products' : 'horizontal-products'}}"></salla-products-list>
```
Comment (salkhwlani) [inline src/views/pages/landing-page.twig:83]:
```html
<salla-products-list source="landing-page"></salla-products-list>```

File: src/views/pages/partials/product/card-mini.twig [hunk 4]
```diff
@@ -1,53 +1 @@
...
-            <salla-add-product-button fill="outline" width="wide"
-                                      product-id="{{ product.id }}"
-                                      product-status="{{ product.status }}"
-                                      product-type="{{ product.type }}">
-                {{ product.add_to_cart_label }}
-            </salla-add-product-button>
-        </div>
-    </div>
-</div>
+<salla-product-card shadow-on-hover product="{{product}}" minimal></salla-product-card>
```
Comment (salkhwlani) [inline src/views/pages/partials/product/card-mini.twig:1]: drop the file, and use this directly on the place

File: src/views/pages/partials/product/card.twig [hunk 5]
```diff
@@ -6,137 +6,4 @@ view options:
...
-                            onclick="salla.wishlist.toggle({{ product.id }})"
-                            data-id="{{ product.id }}">
-                        <i class="sicon-heart text-xl"></i>
-                    </salla-button>
-                {% endif %}
-            </div>
-        {% endif %}
-    </div>
-</div>
+<salla-product-card shadow-on-hover product="{{product}}" {{is_special ? 'is-special' : ''}} {{is_horizontal ? 'horizontal' : ''}}></salla-product-card>
```
Comment (salkhwlani) [inline src/views/pages/partials/product/card.twig:9]: Drop this file and use the component directly on the place

File: src/views/pages/product/index.twig [hunk 6]
```diff
@@ -54,23 +54,7 @@
...
-                        {% endfor %}
-                    </salla-infinite-scroll>
-                {% else %}
-                    <div class="no-content-placeholder">
-                        <i class="sicon-shopping-bag icon"></i>
-                        <p>{{ trans('pages.categories.no_products') }}</p>
-                    </div>
-                {% endif %}
-
+                    <salla-products-list></salla-products-list>
```
Comment (salkhwlani) [inline src/views/pages/product/index.twig:57]:
```html
<salla-products-list source="page.slug" source-value="page.id"></salla-products-list>
```

File: src/views/pages/product/single.twig [hunk 7]
```diff
@@ -380,7 +380,9 @@
     </div>
 
     {% component 'comments' %}
-    {% component 'product.similar-products' %}
+    <div class="container">
+        <salla-products-slider block-title="{{trans('pages.products.similar_products')}}" display-all-url></salla-products-slider>
```
Comment (salkhwlani) [inline src/views/pages/product/single.twig:384]: <salla-products-list source="related" source-value="product.id"></salla-products-list>

File: src/views/pages/partials/product/card-full-image.twig [hunk 8]
```diff
@@ -1,68 +1 @@
...
-                    aria-label="Add or remove to wishlist"
-                    class="btn--wishlist animated"
-                    onclick="salla.wishlist.toggle({{ product.id }})"
-                    data-id="{{ product.id }}">
-                <i class="sicon-heart"></i>
-            </salla-button>
-        </div>
-    </div>
-</div>
+<salla-product-card shadow-on-hover product="{{product}}" full-image></salla-product-card>
```
Comment (salkhwlani) [inline src/views/pages/partials/product/card-full-image.twig:1]: drop the file, and use this directly on the place

**Review by salkhwlani — CHANGES_REQUESTED** [review]

**Comment by SallaDev** [comment]
> The branch `support-product-components` of https://github.com/SallaApp/theme-raed/pull/192 has been synced from master..✅

**Comment by SallaDev** [comment]
> The branch `support-product-components` of https://github.com/SallaApp/theme-raed/pull/192 has been synced from master..✅

**Comment by SallaDev** [comment]
> The branch `support-product-components` of https://github.com/SallaApp/theme-raed/pull/192 has Merge conflict❌

**Review by jalmatari — APPROVED** [review]

**Review by jalmatari — APPROVED** [review]

**Pull request merged**